src/charm.py

```diff
@@ -157,7 +159,7 @@ def _on_traefik_pebble_ready(self, _: PebbleReadyEvent):
                 },
                 # We always start the Prometheus endpoint for simplicity
                 # TODO: Generate this file in the dynamic configuration folder when the
-                # metrics-endpoint relation is established?
+                #  metrics-endpoint relation is established?
```

Comment:
```suggestion
                # metrics-endpoint relation is established?
```

Reply:
that was intentional. Convention is to indent multiline todo's and fixme's to make them count as one. It might be just a pycharm thing however.

Reply:
agreed for consistency with other TODOs/FIXMEs in the file